core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/authorization/DBConstants.java

```diff
@@ -73,6 +75,7 @@
     public static final String DELETE_USER_PERMISSION_SQL = "DELETE FROM UM_USER_PERMISSION WHERE UM_USER_NAME=? " +
             "AND UM_PERMISSION_ID = (SELECT UM_ID FROM UM_PERMISSION WHERE UM_RESOURCE_ID = ? AND " +
             "UM_ACTION = ? AND UM_TENANT_ID=?) AND UM_TENANT_ID=?";
+    public static final String DELETE_USER_PERMISSIONS_BY_TENANT_ID_SQL = "DELETE FROM UM_USER_PERMISSION WHERE UM_TENANT_ID = ?";
```

Comment:
Shall we format the newly added SQL lines to follow the maximum characters per line rule (<=120)? WDYT? Please format the other places as well.